README.md

```diff
@@ -162,7 +162,9 @@ let clientSecret: string
 
 const client = generator('mastodon', BASE_URL)
 
-client.registerApp('Test App')
+client.registerApp('Test App', {
+  scopes: ['read', 'write', 'follow']
+})
```

Comment:
Doesn't it work with below?
```suggestion
client.registerApp('Test App', {})
```

Reply:
I'll have to test to see if it works, but wouldn't it need to know which scopes to allow when registering the app?

Reply:
Of course, it is better to know the scope. However, it is not required because a default scope is defined in megalodon.